source/class/qx/ui/form/AbstractField.js

```diff
@@ -131,7 +131,9 @@ qx.Class.define("qx.ui.form.AbstractField", {
     // since ios allows no other html engines, checking for ios should do
     if (qx.core.Environment.get("os.name") == "ios") {
       el.addListener("blur", () => {
-        this._onChangeContent(new qx.event.type.Data(this.getValue()));
+        let ev = new qx.event.type.Data();
```

Comment:
As before: Should only be fired when content is changed. This is documented here: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event